src/test/java/bio/terra/service/search/SearchServiceTest.java

```diff
@@ -48,19 +46,18 @@
 @RunWith(MockitoJUnitRunner.StrictStubs.class)
 @Category(Unit.class)
 public class SearchServiceTest {
-  private static final String sqlQuery =
-      "SELECT GENERATE_UUID() uuid, CURRENT_TIMESTAMP() AS example_now"
-          + " FROM UNNEST(GENERATE_ARRAY(1, 3));";
-
   private static final String timPropertyName = "example:identifier.now";
   private static final String timEncodedName = TimUtils.encode(timPropertyName);
 
+  private static final String sqlQuery =
```

Comment:
Can you add a test that verifies the output when the input is invalid sql query?

Reply:
What type of invalid SQL query were you thinking of? The BigQuery layer currently does the parsing, so if that is invalid there will be an exception. There are connected tests in BigQueryPdao.